webapp/src/utils/index.ts

```diff
@@ -58,14 +58,16 @@ export const getCreateTaskModalCommandArgs = (arr: Array<string>): CreateTaskCom
     };
 };
 
-export const onPressingEnterKey = (event: React.KeyboardEvent<HTMLSpanElement>, func: () => void) => {
+export const onPressingEnterKey = (event: React.KeyboardEvent<HTMLSpanElement> | React.KeyboardEvent<SVGSVGElement>, func: () => void) => {
```

Comment:
```suggestion
export const onPressingEnterKey = (event: React.KeyboardEvent<HTMLSpanElement> | React.KeyboardEvent<SVGElement>, func: () => void) => {
```
Is the type for element here contains "SVG" two times?

Reply:
yes, I also noticed this but this is expected
![Screenshot from 2022-09-08 15-54-04](https://user-images.githubusercontent.com/72438220/189099192-5c315c76-2428-405f-808b-57d6514535e9.png)